collectors/ebpf.plugin/ebpf.d/cachestat.conf

```diff
@@ -3,15 +3,17 @@
 #  `return : In the `return` mode, the eBPF collector monitors the same kernel functions as `entry`, but also creates
 #            new charts for the return of these functions, such as errors.
 #
-# The eBPF collector also creates charts for each running application through an integration with the `apps plugin`.
-# If you want to disable the integration with `apps.plugin` along with the above charts, change the setting `apps` to
-# 'no'.
+# The eBPF collector also creates charts for each running application through an integration with the `apps plugin`
+# or `cgroup plugin`.
+# If you want to disable the integration with `apps.plugin` or `cgroup` along with the above charts, change the setting 
+# `apps` and `cgroup` to  'no'.
 #
 # The `pid table size` defines the maximum number of PIDs stored inside the application hash table.
 #
 # Uncomment lines to define specific options for thread.
 [global]
 #    ebpf load mode = entry
 #    apps = yes
+#    cgroup = no
```

Comment:
Why do you use `cgroup` instead of `cgroups` in configuration files?

Reply:
I changed for plural, the motive was the fact that on Slackware I have only one cgroup by default. Thanks!